src/number_theory/zeta_function.lean

```diff
@@ -534,3 +539,172 @@ begin
   { rw [ne.def, cpow_eq_zero_iff, not_and_distrib, ←ne.def, of_real_ne_zero],
     exact or.inl (pi_pos.ne') }
 end
+
+lemma zeta_nat_eq_tsum_of_gt_one {k : ℕ} (hk : 1 < k) :
+  riemann_zeta k = ∑' (n : ℕ), 1 / n ^ k :=
```

Comment:
Would this be true for any `s` with `re s > 1`? If so, it might be worth a separate lemma.